docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -90,21 +99,21 @@ This method of provisioning the cluster would act as a single control point for
 - Create the new ClusterClass CRD which can serve as a collection of templates to create clusters.
 - Extend the Cluster object to use ClusterClass for creating managed topologies.
 - Enhance the Cluster object to act as a single point of control for the topology.
-- Extend the Cluster controller to create/update/delete managed topologies.
+- Extend the Cluster controller to create/update/delete managed topologies (this includes continuous reconciliation of the topology managed resources).
+- Introduce mechanisms to allow Cluster-specific customizations of a ClusterClass 
 
 ### Prospective future Work
 
 ⚠️ The following points are mostly ideas and can change at any given time  ⚠️
 
 We are fully aware that in order to exploit the potential of ClusterClass and managed topologies, the following class of problems still needs to be addressed:
-- **Lifecycle of the ClusterClass**: Introduce mechanisms for allowing mutation of a ClusterClass, and the continuous reconciliation of the Cluster managed resources.
 - **Upgrade/rollback strategy**: Implement a strategy to upgrade and rollback the managed topologies.
-- **Extensibility/Transformation**: Introduce mechanism for allowing Cluster specific transformations of a ClusterClass (e.g. inject API Endpoint for CAPV, customize machine image by version etc.)
+- **Extensibility/Transformation**: Extend ClusterClass patch variables by introducing support for complex types (e.g. object, array, map, type composition (e.g. int or string)).
 - **Adoption**: Providing a way to convert existing clusters into managed topologies.
 - **Observability**: Build an SDK and enhance the Cluster object status to surface a summary of the status of the topology.
 - **Lifecycle integrations**: Extend ClusterClass to include lifecycle management integrations such as MachineHealthCheck and Cluster Autoscaler to manage the state and health of the managed topologies.
 
-However we are intentionally leaving them out from this initial iteration for the following reasons:
+However, we are intentionally leaving them out from this initial iteration for the following reasons:
```

Comment:
nit - just a grammar preference, but these unnecessary commas make things harder to parse IMO, even though a bunch of automated tools seem to love them 😐 
```suggestion
However we are intentionally leaving them out from this initial iteration for the following reasons:
```